docs/src/main/resources/content/_using/replication-limitations.adoc

```diff
@@ -1,33 +0,0 @@
-:title: Known Issues, Limitations, and Assumptions
```

Comment:
❓ Does this limitations documentation still have value? These are technically still applicable with master. Should it become part of the README?

Reply:
Everything except for derived resources is a limitation imposed by federation, but that shouldn't really be an issue anymore because they should just be querying and replicating from ION right? I'm not sure if derived resources is an issue either because the derived resources should be the same when generated on either side right?

Reply:
Ya in the future many of these will go away. In the meantime harvesting from existing DDF sites will still have these issues.

Reply:
I included the limitations in the readme